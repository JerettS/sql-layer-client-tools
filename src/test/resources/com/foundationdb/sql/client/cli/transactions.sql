CREATE TABLE t(id INT);
INSERT INTO t VALUES (1);
BEGIN;
INSERT INTO t VALUES (2);
SELECT * FROM t;
ROLLBACK;
SELECT * FROM t;
BEGIN;
UPDATE t SET id=2 WHERE id=1;
COMMIT;
SELECT * FROM t;
