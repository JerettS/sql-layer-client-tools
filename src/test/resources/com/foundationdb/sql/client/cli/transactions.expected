test=> CREATE TABLE t(id INT);
ROWS: 0
test=> INSERT INTO t VALUES (1);
ROWS: 1
test=> BEGIN;
ROWS: 0
test=> INSERT INTO t VALUES (2);
ROWS: 1
test=> SELECT * FROM t;
 id 
----
  1 
  2 
(2 rows)

test=> ROLLBACK;
ROWS: 0
test=> SELECT * FROM t;
 id 
----
  1 
(1 row)

test=> BEGIN;
ROWS: 0
test=> UPDATE t SET id=2 WHERE id=1;
ROWS: 1
test=> COMMIT;
ROWS: 0
test=> SELECT * FROM t;
 id 
----
  2 
(1 row)

test=> 
